packages/now-cli/src/commands/list.js

```diff
@@ -78,11 +77,15 @@ export default async function main(ctx) {
     return 1;
   }
 
-  const debugEnabled = argv['--debug'];
+  const {
+    authConfig: { token },
+    output,
+    apiUrl,
+    config,
+  } = ctx;
 
-  const { print, log, error, note, debug } = createOutput({
-    debug: debugEnabled,
-  });
+  const debugEnabled = argv['--debug'];
```

Comment:
I think you can remove `debugEnabled` since its the same as `debug` below.

Reply:
The `debug` below is a logger function, not the boolean. Though I agree these should be cleaned up to utilize `output.isDebugEnabled()`, but probably in a separate PR. This one is big enough already.